package.json

```diff
@@ -40,6 +40,7 @@
     "winston": "~1.0.0"
   },
   "devDependencies": {
+    "electron-packager": "^5.0.1",
```

Comment:
While semver is great, not all repositories respect it (e.g. some people introduce breaking changes in minors). I prefer to get burned less often by having less volatility in my dependencies.

If we need a new feature, then we will upgrade explicitly.